nc-news-fe/src/components/CommentContent.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import CommentCard from "./CommentCard";
+
+const CommentContent = ({ comments }) => {
+  const [comment, setComment] = useState("");
+  const [showComments, setShowComments] = useState(false);
+
+  return (
+    <section>
+      <form className="d-flex flex-column align-items-center">
+        <div className="d-flex">
+          <label htmlFor="comment">
+            <input
+              type="text"
+              value={comment}
+              onChange={(e) => setComment(e.target.value)}
+            />
+          </label>
+          <button>Add</button>
+        </div>
+      </form>
+      <h2>Most Recent Comment</h2>
+      <button onClick={() => setShowComments(true)}>View All Comments</button>
+      {comments.length > 0 && showComments ? (

```

Comment:
This button is a bit small and tucked away in the corner - would make for a better user experience to make it larger and perhaps centered if that works with the design of your site